frontend/src/App.tsx

```diff
@@ -26,13 +27,55 @@ export const App: React.FC = () => (
 );
 
 const HomeRoute: React.FC = () => (
-    <Suspense fallback="Loading! (TODO)">
+    <APIWrapper>
         <HomePage />
-    </Suspense>
+    </APIWrapper>
 );
 
 const RealmRoute: React.FC<RouteComponentProps<{ path?: string }>> = ({ match }) => (
-    <Suspense fallback="Loading! (TODO)">
+    <APIWrapper>
         <RealmPage path={match.params.path ?? ""} />
-    </Suspense>
+    </APIWrapper>
+);
+
+
+const APIWrapper: React.FC = ({ children }) => (
+    <APIErrorBoundary>
+        <Suspense fallback="Loading! (TODO)">
+            {children}
+        </Suspense>
+    </APIErrorBoundary>
 );
+
+// TODO This is of course rather bare bones;
+// it mainly serves to demonstrate what we have to do to **get to** the
+// errors we might get from Relay, not necessarily how to handle them.
+class APIErrorBoundary extends React.Component<unknown, boolean> {
+    public constructor(props: unknown) {
+        super(props);
+        this.state = false;
+    }
+
+    public static getDerivedStateFromError(error: unknown) {
+        console.dir(error);
```

Comment:
Mh, sure that's better than `console.error`? It would be nice to have something red in the console.

Reply:
Whups, that wasn't even supposed to be there. I used that to test some stuff. I would just remove it now, or do we want to log it somehow? At least in the cases I tested, the error was already logged, but that might not always be the case. Our case is special, because the error happens asynchronously.

Reply:
Uhm... I want to see the error in the console. If that happens automatically, then we don't need to. Otherwise I would want to include it.

Reply:
OK, I tested some more and all errors seem to be reported already, be they sync or async. Removed the debug output.

Reply:
I checked the production build as well and yes, the error is automatically printed, so we don't have to do it.